### DISCUSSION TITLE
Is shields.io down ?

### DISCUSSION THREAD

**Discussion by InstaZDLL** [badges/shields#9172] [Q&A] [answered]
I **can't connect** to **shields.io**, I get this error :
![Screenshot 2023-05-15 131321](https://github.com/badges/shields/assets/72951793/15d19814-fd03-4a36-b4f6-03d5e98a6768)

**Comment by Mqxx** [comment, marked as answer]
[Here](https://stats.uptimerobot.com/PjXogHB5p) you can see the status. 👍

**Reply by InstaZDLL** [reply]
Thanks
